LocalStorage/LocalStorageManager/LocalStorageManagerProtocol.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LocalStorageManagerProtocol.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 06/04/2024.
+//
+
+import Foundation
+import Combine
+
+protocol LocalStorageManagerProtocol {
+    static var shared: LocalStorageManager {get}
```

Comment:
```suggestion
```
Protocol should represent functions not implementation, then this should be removed and will be kept in the concrete implementation.